using_images/xpaas_images/a_mq.adoc

```diff
@@ -105,6 +115,61 @@ application's Git project root. On each commit, the file will be copied to the
 *_conf_* directory in the A-MQ root and its contents used to configure the
 broker.
 
+== Configuring the JBoss A-MQ Persistance Image
+
+[[configuring-params]]
+=== Application Template Parameters
+
+Basic configuration of the JBoss A-MQ Persistance xPaaS image is performed by specifying
+values of application template parameters. The following parameters can be
+configured:
+
+`*AMQ_RELEASE*`::
+  The JBoss A-MQ release version. This determines which JBoss A-MQ image will be
+  used as a basis for the application. At the moment, only version _6.2_ is
+  available.
+`*APPLICATION_NAME*`::
+  The name of the application used internally in OpenShift. It is used in names
+  of services, pods, and other objects within the application.
+`*MQ_PROTOCOL*`::
+  Comma-separated list of the messaging protocols used by the broker. Available
+  options are _amqp_, _mqtt_, _openwire_, and _stomp_. If left empty, all
+  available protocols will be available. Please note that for integration of the
+  image with Red Hat JBoss Enterprise Application Platform, the _openwire_
+  protocol must be specified, while other protocols can be optionally specified
+  as well.
+`*MQ_QUEUES*`::
+  Comma-separated list of queues available by default on the broker on its
+  startup.
+`*MQ_TOPICS*`::
+  Comma-separated list of topics available by default on the broker on its
+  startup.
+`*VOLUME_CAPACITY*`::
+  The size of the persistence storage for database volumes. 
+`*MQ_USERNAME*`::
+  The user name used for authentication to the broker. In a standard
+  non-containerized JBoss A-MQ, you would specify the user name in the
+  *_AMQ_HOME/opt/user.properties_* file. If no value is specified, a random user
+  name is generated.
+`*MQ_PASSWORD*`::
+  The password used for authentication to the broker. In a standard
+  non-containerized JBoss A-MQ, you would specify the password in the
+  *_AMQ_HOME/opt/user.properties_* file. If no value is specified, a random
+  password is generated.
+`*AMQ_ADMIN_USERNAME*`::
+  The username used as an admin authentication to the broker. If no value is specified, a random user name is generated.
+`*AMQ_ADMIN_PASSWORD*`::
+  The password used for authentication to the broker. If no value is specified, a random password is generated.
+`*AMQ_SECRET*`::
+  The name of a secret containing SSL related files. If no value is specified, a random password is generated.
+`*AMQ_TRUSTSTORE*`::
+  The SSL trust store filename. If no value is specified, a random password is generated.
+`*AMQ_KEYSTORE*`::
+  The SSL key store filename. If no value is specified, a random password is generated.
+
+For more information, see 
+link:../../https://github.com/openshift/openshift-docs/blob/master/dev_guide/persistent_volumes.adoc#using-persistent-volumes[Using Persistance Volumes]
```

Comment:
This link does not work. Should be:

```
link:../../dev_guide/persistent_volumes.html#using-persistent-volumes[Using Persistent Volumes].
```

Also note in the above the s/Persistance/Persistent/ and addition of the period.